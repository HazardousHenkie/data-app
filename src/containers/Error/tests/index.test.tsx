import React from 'react'
import renderer from 'react-test-renderer'

import MockingAppComponent from 'utils/testing/MockingAppComponent'
import ErrorPage from '../index'

describe('<ErrorPage />', () => {
    it('should render like snapshot', () => {
        const component = renderer.create(
            <MockingAppComponent>
                <ErrorPage errorCode={404} errorMessage="Page not found" />
            </MockingAppComponent>
        )

        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})
