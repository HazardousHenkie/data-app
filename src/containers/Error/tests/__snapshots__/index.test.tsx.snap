// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorPage /> should render like snapshot 1`] = `
<div
  className="sc-AxirZ guwtUH"
>
  <h1
    className="MuiTypography-root sc-AxjAm cWFipc MuiTypography-h1 MuiTypography-alignCenter"
  >
    404
  </h1>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
  >
    Page not found
  </p>
</div>
`;
